app/pages/studies.py

```diff
@@ -33,7 +34,24 @@
 
 
 def study_show_page(studyId):
-    study = _fetch_study(studyId, check_user_visibility=False)
+    study = _fetch_study(
+        studyId,
+        check_user_visibility=False,
+        sql_options=(
+            sql.orm.selectinload(
+                Study.experiments,
+                Experiment.bioreplicates,
+                Bioreplicate.measurementContexts,
+                MeasurementContext.technique,
+            ),
+            sql.orm.selectinload(
+                Study.experiments,
+                Experiment.bioreplicates,
+                Bioreplicate.measurementContexts,
+                MeasurementContext.measurements,
+            ),
+        )
+    )
```

Comment:
This syntax fetches the study's relationships in batch, rather than one at a time: https://docs.sqlalchemy.org/en/20/orm/queryguide/relationships.html

It's annoyingly verbose, both blocks exist so that the "leaves" of the relationships, "technique" and "measurements" are both loaded. But it is what it is.